> ##[error] Unsupported line end format KeeperSecurity.Commander.installer.yaml

The installer file appears to have CRLF (`\r\n`) style line endings. Can these be replaced with LF (`\n`) style line endings?

--- a/manifests/k/KeeperSecurity/Commander/16.11.21/KeeperSecurity.Commander.installer.yaml
+++ b/manifests/k/KeeperSecurity/Commander/16.11.21/KeeperSecurity.Commander.installer.yaml
@@ -23,6 +23,7 @@ InstallationMetadata:
   DefaultInstallLocation: '%ProgramFiles%\Keeper Commander'
 Installers:
 - Architecture: x64
+
   InstallerUrl: https://github.com/Keeper-Security/Commander/releases/download/v16.11.21/keeper-commander-windows-v16.11.21.exe
   InstallerSha256: 0FBBBDBD1677DCC37C078A8AEAEECD2256D68928686D8ED6BAF74621A8FB721C
 ManifestType: installer

--- a/manifests/k/KeeperSecurity/Commander/16.11.21/KeeperSecurity.Commander.installer.yaml
+++ b/manifests/k/KeeperSecurity/Commander/16.11.21/KeeperSecurity.Commander.installer.yaml
@@ -22,8 +22,7 @@ ElevationRequirement: elevatesSelf
 InstallationMetadata:
   DefaultInstallLocation: '%ProgramFiles%\Keeper Commander'
 Installers:
-- Architecture: x64
-
+- Architecture: x64
   InstallerUrl: https://github.com/Keeper-Security/Commander/releases/download/v16.11.21/keeper-commander-windows-v16.11.21.exe
   InstallerSha256: 0FBBBDBD1677DCC37C078A8AEAEECD2256D68928686D8ED6BAF74621A8FB721C
 ManifestType: installer

